SkimsAUX/plugins/prodMET.cc

```diff
@@ -64,36 +56,27 @@ prodMET::~prodMET() {
 
 bool prodMET::filter(edm::Event& iEvent, const edm::EventSetup& iSetup) {
 
-  edm::Handle<edm::View<pat::MET> > met;
+  edm::Handle<edm::View<reco::MET> > met;
   iEvent.getByLabel(metSrc_, met);
+  edm::Handle<edm::View<reco::MET> > puppiMet;
+  iEvent.getByLabel(puppiMetSrc_, puppiMet);
 
   std::auto_ptr<double> metPtr(new double);
   std::auto_ptr<double> metphiPtr(new double);
-
-  std::auto_ptr<std::vector<double> > metMagUp_ (new std::vector<double>(uncUpList.size(), 0.));
-  std::auto_ptr<std::vector<double> > metPhiUp_ (new std::vector<double>(uncUpList.size(), 0.));
-  std::auto_ptr<std::vector<double> > metMagDown_ (new std::vector<double>(uncDownList.size(), 0.));
-  std::auto_ptr<std::vector<double> > metPhiDown_ (new std::vector<double>(uncDownList.size(), 0.));
```

Comment:
Why are the metMag* and metPhi* removed? We need them for met systematic studies